webapp/channels/src/components/common/hooks/useGetUsage.ts

```diff
@@ -1,52 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import {useEffect, useState} from 'react';
-import {useDispatch, useSelector} from 'react-redux';
-
-import type {CloudUsage} from '@mattermost/types/cloud';
-
-import {isCurrentLicenseCloud} from 'mattermost-redux/selectors/entities/cloud';
-import {getUsage} from 'mattermost-redux/selectors/entities/usage';
-
-import {
-    getMessagesUsage,
-    getFilesUsage,
-    getTeamsUsage,
```

Comment:
Similarly to Daniel's comment about limit actions, should these be removed as well? They also have corresponding `Client4` methods.

I see that we're still dispatching `getTeamsUsage` in response to some WebSocket events. If that's needed still, we should probably file a followup ticket to revisit how that's done since unconditionally firing a network request in response to a WS event could cause thundering herd problems